tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,7 @@
 const App = () => {
-	return <div>A tic-tac-toe game</div>;
+	return (
+	  <GameBoard />
```

Comment:
The application has a separation of concerns by having each component in its own module. This is helpful for keeping code accessible and more easily maintainable. 👍 .

Reply:
Thank you, I always try to keep most module separate incase of any errors, so I know exactly where to go.